GPU/Vulkan/DrawEngineVulkan.cpp

```diff
@@ -962,6 +962,7 @@ void DrawEngineVulkan::DoFlush() {
 
 			// We let the framebuffer manager handle the clear. It can use renderpasses to optimize on tilers.
 			// If non-buffered though, it'll just do a plain clear.
+			// TODO: Don't we need to rescale result.depth if we use GPU_SCALE_DEPTH_FROM_24_TO_16_BIT and similar?
```

Comment:
As noted in the other issue, `result.depth` should already be the scaled value.

-[Unknown]

Reply:
Oh duh.